store/src/java/com/zimbra/cs/account/Provisioning.java

```diff
@@ -634,7 +634,9 @@ public AlwaysOnCluster getAlwaysOnCluster(Server server) throws ServiceException
      */
     public String getEmailAddrByDomainAlias(String emailAddress) throws ServiceException {
         String addr = null;
-
+        if(null == emailAddress){
```

Comment:
formatting issue. Add space after if
```suggestion
        if (null == emailAddress){
```